Automattic/wp-calypso
# Create ActionCard block from select-business cta-card

This PR refactors `CTACard` to a block, called `ActionCard`:

![screen shot 2018-03-19 at 15 05 01](https://user-images.githubusercontent.com/326402/37597093-006ee4c4-2b87-11e8-8974-e8ce0a315568.png)


It is used on Google My Business page:
![screen shot 2018-03-19 at 15 05 51](https://user-images.githubusercontent.com/326402/37597120-15fe38d0-2b87-11e8-8998-de5974519ea9.png)


#### Testing instructions
  
1. Run `git checkout add/action-card` and start your server, or open a [live branch](https://calypso.live/?branch=add/action-card)
2. Open the [`DevDocs` page](http://calypso.localhost:3000/devdocs/design/action-card)
3. Check that you see `ActionCard` as on the screenshot above
4. Make sure that Google My Business select business type page that'll be available at route `/google-my-business/:site_slug` still looking fine.

#### Reviews
